Depending on "cp" version, alsa-utils install may fail with:
	"$(TARGET_DIR)/usr/share/alsa/ directory doesn't exist"
This patch fixes it.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: buildroot/package/multimedia/alsa-utils/alsa-utils.mk
===================================================================
--- buildroot.orig/package/multimedia/alsa-utils/alsa-utils.mk	2011-08-08 12:11:53.000000000 +0200
+++ buildroot/package/multimedia/alsa-utils/alsa-utils.mk	2011-08-08 12:16:40.000000000 +0200
@@ -49,7 +49,7 @@
 	if [ -x "$(TARGET_DIR)/usr/sbin/alsactl" ]; then \
 		mkdir -p $(TARGET_DIR)/usr/share/; \
 		rm -rf $(TARGET_DIR)/usr/share/alsa/; \
-		cp -rdpf $(STAGING_DIR)/usr/share/alsa/ $(TARGET_DIR)/usr/share/alsa/; \
+		cp -rdpf $(STAGING_DIR)/usr/share/alsa $(TARGET_DIR)/usr/share/alsa; \
 	fi
 endef
 
